### DISCUSSION TITLE
The roundstart And the roundend

### DISCUSSION THREAD

**Discussion by jacobxy** [saul/demofile#241] [Questions] [answered]
When I parse a dem file, I see that the count of "roundstart" isn't equal to the count of "roundend".
I don't know why?
And that, the count of "roundend" is much less then the count of match.
I can not get the result of the every small match, because only the message of 'roundend' can show me which team is the winner.

Can you help me?
Thanks

**Comment by saul** [comment, marked as answer]
Is this a GOTV demo recorded on the server, or a demo recorded on a GOTV relay? I don't think the `round_end` event is sent reliably (i.e. guaranteed delivery), and so sometimes is missed from GOTV demos. What _is_ reliable is checking the team score props:

https://github.com/saul/demofile/blob/3ca6b43c594b67c05bbe4d33def5b12a7061366e/examples/dumpfile.js#L54-L61

**Comment by jacobxy** [comment]
Thank you very much!
It works well!
